src/code/code_80097A00.c

```diff
@@ -78,136 +78,141 @@ u32 gGsFlagsMasks[] = { 0x000000FF, 0x0000FF00, 0x00FF0000, 0xFF000000 };
+    gItemIconMaskKeatonTex,          // ITEM_MASK_KEATON
+    gItemIconMaskSkullTex,           // ITEM_MASK_SKULL
+    gItemIconMaskSpookyTex,          // ITEM_MASK_SPOOKY
+    gItemIconMaskBunnyTex,           // ITEM_MASK_BUNNY
+    gItemIconMaskGoronTex,           // ITEM_MASK_GORON
+    gItemIconMaskZoraTex,            // ITEM_MASK_ZORA
+    gItemIconMaskGerudoTex,          // ITEM_MASK_GERUDO
+    gItemIconMaskTruthTex,           // ITEM_MASK_TRUTH
+    gItemIconSoldOutTex,             // ITEM_SOLD_OUT
+    gItemIconPocketEggTex,           // ITEM_POCKET_EGG
+    gItemIconPocketCuccoTex,         // ITEM_POCKET_CUCCO
+    gItemIconCojiroTex,              // ITEM_COJIRO
+    gItemIconOddMushroomTex,         // ITEM_ODD_MUSHROOM
+    gItemIconOddPotionTex,           // ITEM_ODD_POTION
+    gItemIconPoachersSawTex,         // ITEM_POACHERS_SAW
+    gItemIconBrokenBiggoronSwordTex, // ITEM_BROKEN_BGS
+    gItemIconPrescriptionTex,        // ITEM_PRESCRIPTION
+    gItemIconEyeballFrogTex,         // ITEM_EYEBALL_FROG
+    gItemIconEyeDropsTex,            // ITEM_EYE_DROPS
+    gItemIconClaimCheckTex,          // ITEM_CLAIM_CHECK
+    gItemIconBowFireTex,             // ITEM_BOW_ARROW_FIRE
+    gItemIconBowIceTex,              // ITEM_BOW_ARROW_ICE
+    gItemIconBowLightTex,            // ITEM_BOW_ARROW_LIGHT
+    gItemIconSwordKokiriTex,         // ITEM_SWORD_KOKIRI
+    gItemIconSwordMasterTex,         // ITEM_SWORD_MASTER
+    gItemIconSwordBiggoronTex,       // ITEM_SWORD_BIGGORON
+    gItemIconShieldDekuTex,          // ITEM_SHIELD_DEKU
+    gItemIconShieldHylianTex,        // ITEM_SHIELD_HYLIAN
+    gItemIconShieldMirrorTex,        // ITEM_SHIELD_MIRROR
+    gItemIconTunicKokiriTex,         // ITEM_TUNIC_KOKIRI
+    gItemIconTunicGoronTex,          // ITEM_TUNIC_GORON
+    gItemIconTunicZoraTex,           // ITEM_TUNIC_ZORA
+    gItemIconBootsKokiriTex,         // ITEM_BOOTS_KOKIRI
+    gItemIconBootsIronTex,           // ITEM_BOOTS_IRON
+    gItemIconBootsHoverTex,          // ITEM_BOOTS_HOVER
+    gItemIconBulletBag30Tex,         // ITEM_BULLET_BAG_30
+    gItemIconBulletBag40Tex,         // ITEM_BULLET_BAG_40
+    gItemIconBulletBag50Tex,         // ITEM_BULLET_BAG_50
+    gItemIconQuiver30Tex,            // ITEM_QUIVER_30
+    gItemIconQuiver40Tex,            // ITEM_QUIVER_40
+    gItemIconQuiver50Tex,            // ITEM_QUIVER_50
+    gItemIconBombBag20Tex,           // ITEM_BOMB_BAG_20
+    gItemIconBombBag30Tex,           // ITEM_BOMB_BAG_30
+    gItemIconBombBag40Tex,           // ITEM_BOMB_BAG_40
+    gItemIconGoronsBraceletTex,      // ITEM_STRENGTH_GORONS_BRACELET
+    gItemIconSilverGauntletsTex,     // ITEM_STRENGTH_SILVER_GAUNTLETS
+    gItemIconGoldenGauntletsTex,     // ITEM_STRENGTH_GOLD_GAUNTLETS
+    gItemIconScaleSilverTex,         // ITEM_SCALE_SILVER
+    gItemIconScaleGoldenTex,         // ITEM_SCALE_GOLDEN
+    gItemIconBrokenGiantsKnifeTex,   // ITEM_SWORD_KNIFE
```

Comment:
Here.

Reply:
Not sure if the item should say "broken", it's a weird case and I can't find definitive code to understand it